code/game/objects/items/documents.dm

```diff
@@ -69,19 +74,35 @@
 	w_class = WEIGHT_CLASS_TINY
 	throw_range = 1
 	throw_speed = 1
+	///time required to print a report
+	var/print_time = 5 SECONDS
 
 /obj/item/inspector/attack_self(mob/user)
 	. = ..()
-	if(do_after(user, 5 SECONDS, target = user, progress=TRUE))
+	if(do_after(user, print_time, target = user, progress=TRUE))
 		print_report()
 
-///Prints out a report for bounty purposes, and plays a short audio blip.
-/obj/item/inspector/proc/print_report()
-	// Create our report
+/**
+ * Create our report
+ *
+ * Arguments:
+ */
+/obj/item/inspector/proc/create_slip()
 	var/obj/item/paper/report/slip = new(get_turf(src))
 	slip.generate_report(get_area(src))
+
+/obj/item/inspector/proc/play_sound()
```

Comment:
As far as I can tell you never call only this proc anywhere and there's not really a reason for it to exist, why did you separate this into its own proc?

Reply:
I was trying to do something that I later realized was stupid.